src/lib/y2partitioner/format_mount/options.rb

```diff
@@ -23,47 +23,6 @@
 require "y2storage"
 
 module Y2Storage
-  # FIXME: Monkey patch of Y2Storage::PartitionId it should moved to
```

Comment:
well, I do not increase version of yast2-storage-ng, so rpm dependency does not change. Just ensure you have the latest one.